cupy/testing/helper.py

```diff
@@ -635,6 +636,14 @@ def check_func(x, y):
                            accept_error, sp_name, scipy_name)
 
 
+def _has_ndarray(x):
+    if isinstance(x, (numpy.ndarray, cupy.ndarray)):
+        return True
+    if isinstance(x, collections.abc.Iterable):
```

Comment:
```suggestion
    if isinstance(x, collections.abc.Sequence):
```
`Sequence` looks better because iterators can be consumed.

Use `(tuple, list)` to be compatible with other decorators.